contents/blog/posthog-vs-matomo.mdx

```diff
@@ -19,321 +19,194 @@ import { ComparisonTable } from 'components/ComparisonTable'
-  <ComparisonRow column1={false} column2={true} feature="Google Analytics" />
-  <ComparisonRow column1={true} column2={false} feature="API" />
-  <ComparisonRow column1={true} column2={false} feature="Redshift" />
-  <ComparisonRow column1={true} column2={false} feature="Google Cloud Storage" />
-  <ComparisonRow column1={true} column2={false} feature="Snowflake" />
-  <ComparisonRow column1={true} column2={false} feature="Amazon S3" />
-  <ComparisonRow column1={true} column2={false} feature="Sentry" />
-  <ComparisonRow column1={true} column2={false} feature="Hubspot" />
-  <ComparisonRow column1={true} column2={false} feature="Salesforce" />
-  <ComparisonRow column1={true} column2={false} feature="Zapier" />
-  <ComparisonRow column1={true} column2={false} feature="Zendesk" />
+ <ComparisonRow column1={true} column2={true} feature="Product analytics" description="Track events and conversion; analyze user behavior" />
+  <ComparisonRow column1={true} column2={true} feature="Web analytics" description="Simple dashboard of aggregate traffic, sources, campaigns, and session metrics" />
+  <ComparisonRow column1={true} column2="Add on" feature="Session replay" description="Watch real users use your product, diagnose bugs" />
+	<ComparisonRow column1={true} column2="Add on" feature="Heatmaps" description="Visualize where users click in your app or website" />
+  <ComparisonRow column1={true} column2="Add on" feature="Experiments" description="Run A/B/n tests on new features, optimize conversion funnels" />
+  <ComparisonRow column1={true} column2={false} feature="Surveys" description="Collect and analyze feedback, run NPS and PMF surveys" />
+  <ComparisonRow column1={true} column2={false} feature="Feature flags" description="Roll out features safely, toggle features for cohorts or individuals" />
+	<ComparisonRow column1={true} column2={false} feature="Free tier" description="Free forever option, no credit card required" />
+	<ComparisonRow column1="Unlimited" column2="30" feature="Free team members" description="Number of users you can invite to your instance" />
+	<ComparisonRow column1={true} column2={true} feature="EU hosting" description="Access and store your data in the EU" />
+  <ComparisonRow column1={true} column2={true} feature="Open source" description="View code, comment on roadmap, and contribute" />
+  <ComparisonRow column1={true} column2={true} feature="Server-side SDKs" description="Capture events and use features from Python, Node, and more" />
+  <ComparisonRow column1={true} column2={true} feature="Mobile SDKs" description="Capture events and use features from Android, iOS, and more" />
+  <ComparisonRow column1={true} column2={true} feature="API" description="Capture events, get stats, and make changes via API" />
 </ComparisonTable>
 
-## Strengths of PostHog
+> 💡 **Good to know:** Matomo's add owns can cost hundreds of dollars per year. For example, their "Heatmap & Session Recording" add on starts at 219€ per year.
```

Comment:
This on-prem only. I think we should stick to comparing cloud vs cloud.

Reply:
It was confusing, now using their pricing page as source of truth.